setup/stability-extra/multicluster-vpn/default/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo
+echo " [*] Installing the 'default' scenario"
+echo
+
+SCENARIO="default"
+NAMESPACE="env-$SCENARIO"
+$KUBECTL2 create ns $NAMESPACE || true
+
+$KUBECTL2 -n $NAMESPACE delete secrets -l "istio/multiCluster=true"
+install_k8s_secrets "$KUBECONFIG2" "$KUBECONTEXT2" "$NAMESPACE" "$KUBECONFIG1" "$KUBECONTEXT1" "istio-system"
+
+cat > ${TEMP_DIR}/values-${SCENARIO}.yaml <<EOF
+global:
+  hub: ${HUB}
+  tag: ${TAG}
+  istioNamespace: ${NAMESPACE}
+  configNamespace: ${NAMESPACE}
+  telemetryNamespace: ${NAMESPACE}
+  policyNamespace: ${NAMESPACE}
+  configValidation: false
+  proxy:
+    tracer: ""
+pilot:
+  configNamespace: ${NAMESPACE}
+EOF
+
+helm template \
```

Comment:
I would prefer if the iop script from installer was used ( which will be replaced at some point with istioctl ) - or just use kustomize without helm.